rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -147,6 +147,13 @@ def _calculate_unclaimed_reward(
                 token_ethereum_address = deserialize_ethereum_address(
                     entry['channelArgs']['tokenAddr'],
                 )
+                try:
+                    token_ethereum_address = to_checksum_address(token_ethereum_address)
```

Comment:
Eventually this things needs to be moved to its own deserialize function.

We already got one funnily enough but it's not good enough:

```python
def deserialize_ethereum_address(symbol: str) -> ChecksumEthAddress:
    return ChecksumEthAddress(HexAddress(HexStr(symbol)))
```

It's just for typing and nothing more.